### PR TITLE
Add descwl-shear-sims package

### PR DESCRIPTION
**Pull request by mr-superonion**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/descwl-shear-sims/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{% set name = "descwl-shear-sims" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
```
Comment (beckermr) [inline recipes/descwl-shear-sims/meta.yaml:5]: We've been taking the convention that DESC software on conda-forge should be prefixed with `lsstdesc-`. This helps with discoverability, namespace issues, etc. 

What do you all think we should do here?

File: recipes/descwl-shear-sims/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  home: https://github.com/LSSTDESC/descwl-shear-sims/
+  license: BSD-3-Clause
+  license_file: LICENSE
+  summary: "DESC weal lensing shear simulations"
+  description: |
+    descwl-shear-sims is a open-source package to simulate galaxy images for
+    shear related tests
+extra:
+  recipe-maintainers:
+    - mr-superonion
```
Comment (beckermr) [inline recipes/descwl-shear-sims/meta.yaml:46]:
```suggestion
    - mr-superonion
    - beckermr
```

**Review by beckermr — CHANGES_REQUESTED** [review]
We should add @esheldon if we wants to be on the recipe. He should also comment on the naming of the package above.

File: recipes/descwl-shear-sims/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  host:
+    - python
+    - setuptools >=38
+    - pip
+  run:
+    - python
+    - numba
+    - galsim
+    - lsstdesc.weaklensingdeblending
+    - stackvana
```
Comment (beckermr) [inline recipes/descwl-shear-sims/meta.yaml:28]: Having this in run would exclude someone using a stack not in conda-forge. We'll need to think about how to handle that.

File: recipes/descwl-shear-sims/meta.yaml (hunk 1 above)
Comment (esheldon) [reply recipes/descwl-shear-sims/meta.yaml:5]: I think `lsstdesc-descwl-shear-sims` is redundant and awkward

Comment (esheldon) [reply recipes/descwl-shear-sims/meta.yaml:5]: I think its OK.  The main confusion is it doesn't match the package name.

File: recipes/descwl-shear-sims/meta.yaml (hunk 3 above)
Comment (beckermr) [reply recipes/descwl-shear-sims/meta.yaml:28]: Conda won't recognize that.

File: recipes/descwl-shear-sims/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/LSSTDESC/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 5c99d9cd8c6714d95e9f49232a0c10283c19ecc2ff2c3c1138818fafd5b868c5
+
+
+build:
+  number: 0
+  skip: True  # [py<310 or py>311 or win]
```
[the remark this answers is not in the document]
Comment (beckermr) [reply recipes/descwl-shear-sims/meta.yaml:15]: We can ignore the windows failures. Noarch only has to pass on Linux.

File: recipes/descwl-shear-sims/meta.yaml (hunk 1 above)
Comment (beckermr) [reply recipes/descwl-shear-sims/meta.yaml:5]: What about `lsstdesc-wl-shear-sims`?

File: recipes/descwl-shear-sims/meta.yaml (hunk 3 above)
Comment (beckermr) [reply recipes/descwl-shear-sims/meta.yaml:27]: That is good for now. I think maybe I will add an external build of stackvana that is a shell package that folks can use against a custom DM env.

File: recipes/descwl-shear-sims/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,46 @@
+{% set version = "0.4.1" %}
+
+package:
+  name: lsstdesc-wl-shear-sims
+  version: {{ version }}
+
+source:
+  url: https://github.com/LSSTDESC/descwl-shear-sims/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 5c99d9cd8c6714d95e9f49232a0c10283c19ecc2ff2c3c1138818fafd5b868c5
+
```
Comment (beckermr) [inline recipes/descwl-shear-sims/meta.yaml:10]:
```suggestion
```

**Pull request merged**